opensrp-chw-core/src/test/java/org/smartregister/chw/core/interactor/NavigationInteractorTest.java

```diff
@@ -0,0 +1,23 @@
+package org.smartregister.chw.core.interactor;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.MockitoAnnotations;
+import org.smartregister.chw.core.BaseUnitTest;
+import org.smartregister.chw.core.utils.CoreConstants;
+
+public class NavigationInteractorTest extends BaseUnitTest {
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void testCount(){
```

Comment:
Kindly follow the test naming convention `test[MethodName][ExpectedResult][Parameters]`